drupal/web/themes/custom/covid_theme/templates/user-login-form.html.twig

```diff
@@ -0,0 +1,14 @@
+{{ form.form_build_id }}
+{{ form.form_id }}
+{{ form.form_token }}
+
+
+{{ form.name }}
+{{ form.pass }}
+
+<div class="one-time-password">
+  {{ form.one_time_password }}
+  <span>Vyplňte pouze v případě využití dvoufaktorového ověření.</span>
```

Comment:
Stringy prosím takhle, ať jsou přeložitelné přes UI.
{{ 'Vyplňte pouze v případě využití dvoufaktorového ověření.'|t }}

Reply:
To bych to tam musel dát přece ale v angličtině, ne? Plus máme v plánu tuhle stránku někdy mít dostupnou anglicky?